graph_ui/js/angular-vis.js

```diff
@@ -2,10 +2,9 @@ angular.module('ngVis', [])
 
     .factory('VisDataSet', function () {
         'use strict';
-        return function (data, options) {
-            // Create the new dataSets
+        return{ Draw: function (data, options) {
             return new vis.DataSet(data, options);
-        };
+        }};
```

Comment:
This is an afterthought since it's already merged:

I am curious as to why `angular-vis.js` is being modified, rather than making changes with our own code. It seems that the only substantial change is in this return call for the VisDataSet factory. Why is this change in here instead of using the old VisDataSet code that came with the file?